README.md

```diff
@@ -124,39 +124,74 @@ a proxy tool such as [mitmproxy](https://mitmproxy.org), or even just
 a script.
 
 A particularly simple way to proxy requests to a private bitcoind node
-is to make use of the [`corsproxy`](https://www.npmjs.com/package/corsproxy)
```

Comment:
I'd like to keep the `corsproxy` solution too, maybe with a `deprecated` header in the title and/or "Preferred" next to the nginx solution. nginx is definitely the safer approach but it's also a little more technical, so it would be nice to have the less technical option available for those willing to take the risk (it does still work at least), especially if it's just to test. Even better would be a more recently updated alternative, but the rolling your own nginx proxy should be preferred anyway to avoid external deps, so I'm happy just marking this as deprecated if you don't know of an alternative off hand.

Reply:
I'll move the `corsproxy` into a deprecated section. At this time, I don't know of any other alternatives. What prompted me to go through this was the reported experience of @mattderron@twitter:
<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Want a stateless <a href="https://twitter.com/hashtag/bitcoin?src=hash&amp;ref_src=twsrc%5Etfw">#bitcoin</a> multi-sig solution that preserves your privacy and/or acts as a permissionless, self-sovereign backup?<br><br>I set up <a href="https://twitter.com/unchainedcap?ref_src=twsrc%5Etfw">@unchainedcap</a> Caravan to be self-hosted pointing to my own node - on Mac.<br><br>A thread with the steps, including errors and how I fixed 👇👇👇</p>&mdash; Matt Derron (@mattderron) <a href="https://twitter.com/mattderron/status/1353756391878635520?ref_src=twsrc%5Etfw">January 25, 2021</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
 
;TLDR he had to roll back the installation of `node` to version 12 to get `corsproxy` to work. Even for testing this is a bit extreme.

Reply:
Oh yeah! I saw that and remember the node version issues. I actually commented on one of the tweets about using nvm for node version control. Much easier to use different versions in different environments which helps somewhat, but your solution of course sidesteps the need for that entirely :).